go.mod

```diff
@@ -346,6 +363,7 @@ replace (
 	github.com/mattn/go-isatty => github.com/mattn/go-isatty v0.0.4
 	github.com/mattn/go-runewidth => github.com/mattn/go-runewidth v0.0.2
 	github.com/mattn/go-shellwords => github.com/mattn/go-shellwords v1.0.5
+	github.com/mattn/go-sqlite3 => github.com/mattn/go-sqlite3 v1.10.0
```

Comment:
this is a lot of random deps to pick up :-/

I know we're not actually depending on all these packages, but their preferred versions of transitive deps still affect version resolution :-/